src/python/bot/fuzzers/options.py

```diff
@@ -157,20 +158,32 @@ def get_ubsan_options(self):
     return self._get_option_section('ubsan')
 
 
+def get_fuzz_target_support_file(fuzz_target_path, file_extension):
```

Comment:
It's probably best to inline this back into the other function. I suspect we won't be adding new files later based on the discussions in this PR.

Reply:
Done